apps/challenges/challenge_config_utils.py

```diff
@@ -177,3 +198,389 @@ def is_challenge_phase_split_mapping_valid(phase_ids, leaderboard_ids, dataset_s
         else:
             return False
     return False
+
+
+def get_value_from_field(data, base_location, field_name):
+    file_path = join(base_location, data.get(field_name))
```

Comment:
what about exception handling for missing values for a field name?

Reply:
If the value is missing, this function will return None. In accordance with the validation. (This is also what happens in the create challenge API)